### ISSUE TITLE
Cropped username at some random points

### ISSUE THREAD

**Issue by DmitriyTrt** [hultmann/multifox#84]
Username is cropped at some random points. See screenshot: http://storage2.static.itmages.com/i/12/1006/h_1349531408_8029521_0fbeb7c9bf.png There should be my email, but instead it shows only a part of it, not even a part before @ sign, it cuts even more. It happens with simple logins too. For example, instead of "DmitriyTrt" I see "DmitriyT". Is it a bug or a feature?

Using Multifox 2.0b5 with Iceweasel 15.0.1 on Debian Linux.

**Comment by hultmann** [comment]
It is a bug, indeed. Are you using the default theme?

**Comment by DmitriyTrt** [comment]
Yes, I'm using default Firefox theme with Nodoka Rounded GTK2 theme on MATE desktop environment (GNOME2 fork).

**Comment by hultmann** [comment]
Could you test with another theme? like https://addons.mozilla.org/en-US/firefox/addon/fxchrome/

**Comment by DmitriyTrt** [comment]
Same bug with this theme. http://storage9.static.itmages.com/i/12/1019/h_1350597339_4606016_dadc821d33.png

**Comment by hultmann** [comment]
Could you send me the URL where you enter your username/password?

**Comment by DmitriyTrt** [comment]
For example, http://gmail.com

**Comment by hultmann** [comment]
Could you copy&paste a multifox domain from your cookies window? http://i.imgur.com/GKAHM.png

P.S. it contains your username, feel free to email me instead (hultmann*gmail)

**Comment by hultmann** [comment]
BTW, is the username cropped in the popup menu?

**Comment by hultmann** [comment]
I used the data you sent me but I still cannot reproduce it. You username is being displayed as expected under Ubuntu and Windows.

Maybe an extension conflict could be causing this problem...

**Comment by DmitriyTrt** [comment]
I'm 100% sure it's not a conflict with other extension, because this bug is reproducible on clean profile with only Multifox installed and enabled.

**Comment by hultmann** [comment]
Maybe a GPU driver bug triggered by linear-gradient CSS...
